pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -100,6 +102,34 @@ func (r *MigPlan) GetDestinationCluster(client k8sclient.Client) (*MigCluster, e
 	return GetCluster(client, r.Spec.DestMigClusterRef)
 }
 
+// GetSourceIdentity - Get the corresponding user for a given source identity
+// token
+// Returns "" if the srcToken doesn't authenticate
+func (r *MigPlan) GetSourceIdentity(client k8sclient.Client) (string, error) {
+	secret, err := r.GetIdentitySecret(client)
+	if err != nil {
+		return "", err
+	}
+	tokenReview := v1beta1.TokenReview{
```

Comment:
I'm not convinced that authenticating the token each time a controller _gets_ it is a good idea.  The cluster token is only authenticated during validation (reconcile) and reports as a condition.  Seems like code using the token will just try to use it and stack trace either way.  Not sure it's worth the additional overhead.

Reply:
The more I think about it, is tokenReview even necessary? I thought it was due to using `SubjectAccessReview`, but in my code refactor it seems that `SelfSubjectAccessReview` allows us to simply use the token in the client without needing to determine the originating user/sa.

My current implementation isn't using it, so I believe we don't need to update any operator RBAC rules unless someone sees a need to use it.